natgeo/mortar
# [SYTO-982] Center Pod Promo and update max-width to be consistent

https://jira.natgeo.com/browse/SYTO-982

This story adds margin: 0 auto; to center the component and updates the max-width to match the AC

To test:

- Pull SYTO-982
- Run npm test to verify tests/linting still pass
- Run gulp serve and verify layout of POD Component
- Hit the merge button
